packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -371,6 +371,10 @@ const Services = WebexPlugin.extend({
    * @returns {object} - User Information including user preferrences .
    */
   getMeetingPreferences() {
+    if (this.webex?.config?.services?.discovery?.wbxapi) {
+      WEBEXAPI = this.webex.config.services.discovery.wbxapi;
```

Comment:
we use wbxappapi in other places also , can we test the whole call flow

Reply:
sorry ya they are same


webex-appapi-service
: 
"https://nebulaac.webex.com/wbxappapi/v1"

@Neeraj-swarnkar  does starting adhoc meeting use the same service !?

Reply:
[Arun](spark-mention://344ea183-9d5d-4e77-aed2-c4f004afd915) wbxappapi and [webexapi.com](https://webexapi.com/) both are different. me and [Kesava](spark-mention://8093d335-9b96-4f9d-a6b2-7293423be88a) we tested this in call together,
We have endpoints correct for integration requests, we didn't see any error for meetingPreference or meetingInfo. It seems there's some error we are getting in client side while creating conversationURL.

Reply:
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-376401 
Ticket for Fixing the USM Adhoc Meeting in an integration environment

cc - @arun3528